PerimeterXModule/Internals/CredentialsIntelligence/PxLoginData.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Web;
+using Jil;
+using PerimeterX.CustomBehavior;
+
+namespace PerimeterX
+{
+    public class PxLoginData
+    {
+        private ICredentialsIntelligenceProtocol protocol;
+        private List<ExtractorObject> loginCredentialsExtractor;
```

Comment:
The names here feel a little off to me. Why just one extractor when it's a list? It should be `loginCredentialsExtractors`, no?